src/AbstractMetricsManager.h

```diff
@@ -314,13 +314,12 @@ class AbstractMetricsManager
     }
 
 public:
-    AbstractMetricsManager(const Configurable *window_config)
+    AbstractMetricsManager(const Configurable *window_config, const std::bitset<64> groups = std::bitset<64>())
```

Comment:
need to add default here because I was getting problem during build because window merge logic